src/test/javascript/spec/component/orion/orion-code-editor-instructor-and-editor-container.component.spec.ts

```diff
@@ -57,41 +61,46 @@ describe('CodeEditorInstructorAndEditorOrionContainerComponent', () => {
                 orionConnectorService = TestBed.inject(OrionConnectorService);
             });
     });
+
     afterEach(() => {
-        sinon.restore();
+        jest.restoreAllMocks();
     });
 
     it('applyDomainChange should inform connector', () => {
-        const selectRepositorySpy = spy(orionConnectorService, 'selectRepository');
+        const selectRepositorySpy = jest.spyOn(orionConnectorService, 'selectRepository');
 
         // @ts-ignore
         comp.applyDomainChange({}, {});
 
-        expect(selectRepositorySpy).to.have.been.calledOnceWithExactly(REPOSITORY.TEST);
+        expect(selectRepositorySpy).toHaveBeenCalledTimes(1);
+        expect(selectRepositorySpy).toHaveBeenCalledWith(REPOSITORY.TEST);
     });
```

Comment:
```suggestion
    });
    
```

Reply:
Thanks, I wasn't aware this is the far more common styling in Artemis